StripeFinancialConnections/StripeFinancialConnections/Source/API Bindings/Models/FinancialConnectionsOAuthPrepane.swift

```diff
@@ -0,0 +1,73 @@
+//
+//  FinancialConnectionsOAuthPrepane.swift
+//  StripeFinancialConnections
+//
+//  Created by Krisjanis Gaidis on 1/6/23.
+//
+
+import Foundation
+@_spi(STP) import StripeUICore
+
+struct FinancialConnectionsOAuthPrepane: Decodable {
+
+    let institutionIcon: FinancialConnectionsImage?
+    let title: String
+    let body: OauthPrepaneBody
+    let partnerNotice: OauthPrepanePartnerNotice?
+    let cta: OauthPrepaneCTA
+    let dataAccessNotice: FinancialConnectionsDataAccessNotice
+
+    struct OauthPrepaneBody: Decodable {
+        let entries: [OauthPrepaneBodyEntry]?
+
+        struct OauthPrepaneBodyEntry: Decodable {
+
+            enum EntryType: String, SafeEnumCodable, Equatable {
+                case text
+                case image
+                case unparsable
+            }
+
+            let type: EntryType
+            private let content: Any?
+            var text: String? {
+                return content as? String
+            }
+            var image: FinancialConnectionsImage? {
+                return content as? FinancialConnectionsImage
+            }
+
+            enum CodingKeys: String, CodingKey {
+                case type
+                case content
+            }
+
+            init(type: EntryType, content: Any?) {
+                self.type = type
+                self.content = content
+            }
+
+            init(from decoder: Decoder) throws {
+                let values = try decoder.container(keyedBy: CodingKeys.self)
+                self.type = try values.decode(EntryType.self, forKey: .type)
+                if let text = try? values.decode(String.self, forKey: .content) {
+                    self.content = text
+                } else if let image = try? values.decode(FinancialConnectionsImage.self, forKey: .content) {
+                    self.content = image
+                } else {
+                    self.content = nil
+                }
+            }
+        }
+    }
+
+    struct OauthPrepanePartnerNotice: Decodable {
+        let partnerIcon: FinancialConnectionsImage?
+        let text: String
+    }
+
+    struct OauthPrepaneCTA: Decodable {
+        let text: String
+        let icon: FinancialConnectionsImage?
+    }
```

Comment:
backend for double-check:
- https://livegrep.corp.stripe.com/view/stripe-internal/pay-server/lib/bank_connections/text/data/oauth_prepane_cta.rb#L7-8